msteams-platform/bots/how-to/get-teams-context.md

```diff
@@ -165,7 +169,22 @@ Response body
     "givenName": "Larry",
     "surname": "Brown",
     "email": "Larry.Brown@fabrikam.com",
-    "userPrincipalName": "labrown@fabrikam.com"
+    "userPrincipalName": "labrown@fabrikam.com",
+    "tenantId":"72f988bf-86f1-41af-91ab-2d7cd011db47", 
+    "userRole":"user"
+}
+```
+Below is response for anonymous user
+
+```http
+GET /v3/conversations/19:ja0cu120i1jod12j@skype.net/members/<anonymous user id>"
```

Comment:
As all other example payloads contain actual values (ex: GET /v3/conversations/19:ja0cu120i1jod12j@skype.net/members/**labrown@fabrikam.com**), I think it will be better to make the anonymous user example to be an actual value as well for consistency purpose. 
ex: GET /v3/conversations/19:ja0cu120i1jod12j@skype.net/members/29:1GcS4EyB_oSI8A88XmWBN7NJFyMqe3QGnJdgLfFGkJnVelzRGos0bPbpsfJjcbAD22bmKc4GMbrY2g4JDrrA8vM06X1-cHHle4zOE6U4ttcc